src/renderer/components/dock/edit-resource/view.tsx

```diff
@@ -101,14 +100,15 @@ class NonInjectedEditResource extends React.Component<EditResourceProps & Depend
     return this.props.editResourceStore.getData(this.tabId);
   }
 
-  async save({ resource, store, tabData }: SaveDraftArgs) {
+  async save({ resource, store }: SaveDraftArgs) {
     if (this.error) {
       return null;
     }
 
-    const currentVersion = yaml.load(this.draft);
-    const firstVersion = yaml.load(tabData.firstDraft ?? this.draft);
-    const patches = createPatch(firstVersion, currentVersion);
+    const currentVersion = yaml.load(this.configurationWhenOpened);
+    const firstVersion = yaml.load(this.configurationInView ?? this.configurationWhenOpened);
+    const patches = createPatch(currentVersion, firstVersion);
```

Comment:
Important change here. It seems that #5195 changed the meaning of these variables, which mean that the order was changed.